src/App.js

```diff
@@ -2,104 +2,41 @@ import React, { Component } from 'react';
 import {
   Box,
   Button,
-  Collapsible,
-  Heading,
   Grommet,
   Layer,
-  ResponsiveContext,
 } from 'grommet';
-import { FormClose, Notification } from 'grommet-icons';
-
-const theme = {
-  global: {
-    colors: {
-      brand: '#228BE6',
-    },
-    font: {
-      family: 'Roboto',
-      size: '14px',
-      height: '20px',
-    },
-  },
-};
-
-const AppBar = (props) => (
-  <Box
-    tag='header'
-    direction='row'
-    align='center'
-    justify='between'
-    background='brand'
-    pad={{ left: 'medium', right: 'small', vertical: 'small' }}
-    elevation='medium'
-    style={{ zIndex: '1' }}
-    {...props}
-  />
-);
 
 class App extends Component {
-  state = {
-    showSidebar: false,
-  }
+  state = { showSidebar: true }
   render() {
     const { showSidebar } = this.state;
     return (
-      <Grommet theme={theme} full>
-        <ResponsiveContext.Consumer>
-          {size => (
-            <Box fill>
-              <AppBar>
-                <Heading level='3' margin='none'>My App</Heading>
+      <Grommet>
+        <Box>
+          <Button
+            label='Toggle Sidebar'
+            type='submit'
+            onClick={() => this.setState({ showSidebar: !this.state.showSidebar })}
+          />
+        </Box>
+        <Box>
+          {showSidebar && (
+            <Layer position='right' full='vertical' modal={false}>
+              <Box
+                fill
+                background='light-2'
+                align='center'
+                justify='center'
+              >
+                Sidebar content
                 <Button
-                  icon={<Notification />}
-                  onClick={() => this.setState({ showSidebar: !this.state.showSidebar })}
+                  type='submit' /* DE1872 */
```

Comment:
When type='button' everything is fine, but when it is 'submit', the slide animation is in the wrong direction, on ie11 only.  Edge, Chrome, etc. are fine.